test/acceptance/tools/deploy_pep.py

```diff
@@ -36,7 +36,7 @@ def set_variables_config(host_proxied_ip, host_proxied_port, port_listening,
                          log_level,
                          plug_in, plug_in_extract_action,
                          bypass_activation='false', bypass_id='',
-                         cache_users='1000', cache_projects='1000', cache_roles='60'):
+                         cache_users='1000', cache_projects='1000', cache_roles='60', administration_port='11211'):
```

Comment:
Answered in https://github.com/telefonicaid/fiware-orion-pep/pull/189#discussion_r25626459

NTC